repo.go

```diff
@@ -118,6 +137,10 @@ func (r repo) language() string {
 		return ""
 	}
 
+	if ok := r.isGithubRemote(); !ok {
```

Comment:
Should be `if !r.isGithubRemote() {`

Reply:
Also this block should be moved to the top of `language()` before `orgRepo = ...`.

Reply:
my bad, was tired when putting this together 😄

Reply:
Also, I would not be able to move this if I included a specific check for the org repo 😛. I think it should stay where it is for now. Once I update the PR, you can have another look.